src/Test/MakerTestEnvironment.php

```diff
@@ -347,22 +340,14 @@ private function buildFlexSkeleton()
                 'replace' => '    <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors" />'."\n".'    </php>',
             ],
         ];
+        $this->processReplacements($replacements, $this->flexPath);
+        // end of temp code
```

Comment:
temp code? :)

Reply:
i can remove this comment if you don't like it =) it is the end of code that we need to remove when deprecation notice goes away :)